.secrets.baseline

```diff
@@ -106,7 +106,7 @@
       }
     ]
   },
-  "version": "0.13.1+ibm.55.dss",
+  "version": "0.13.1+ibm.50.dss",
```

Comment:
@daniel-butler-irl Did you locally run against an old common-dev-assets version? This is reverting the version of DS in main branch

Reply:
the pipeline is failing due to this

Reply:
Must have, although was certain I had updated it. Looking to resolve now